katdal/categorical.py

```diff
@@ -45,9 +45,10 @@ class HashableValueWrapper(object):
     Note
     ----
     The current implementation uses pickle to make generic objects hashable,
-    even though it is not recommended (see http://bit.ly/2ruNkH2). The main
-    problem is with unordered data like dicts and sets, where the item order
-    in the pickle string may differ even though the objects are equal. E.g.
+    even though it is not recommended - see e.g.
+    http://www.aminus.org/blogs/index.php/2007/11/03/pickle_dumps_not_suitable_for_hashing
```

Comment:
Missing full stop at the end of the sentence. This could also be a RST link rather than inline.